docs/source/manifest_submission.rst

```diff
@@ -0,0 +1,274 @@
+     - This can be a **Synapse Project ID** or a **Folder ID**.
+
+   - **asset_view**: The **Synapse ID of the fileview** containing the top-level dataset for which you want to generate a manifest.
+
+   - remove default inputs in **dataset_scope** and **project_scope**
+
+   - set file_annotations_upload to false
+
+   - table_manipulation is "replace" by default. You could keep it that way.
+
+   - set **manifest_record_type** to "file_only" or you could change it based on your project requirements
+
+   - table_column_names: This is optional, and the available options are "class_label", "display_label", and "display_name". The default is "class_label".
+
+
+
+Submit a Manifest file and Add Annotations
+-------------------------------------------
+
+.. note::
+
+  Since annotations are enabled in the submission, if you are submitting a file-based manifest, you should see annotations attached to the entity IDs listed in the manifest.
+
+
+
+Option 1: Use the CLI
+~~~~~~~~~~~~~~~~~~~~~~
+
+
+.. note::
+
+    During submission, validation is optional. If you have finished validation in previous step, you could skip validation by removing `-vc <your data type>`
+
+
+.. code-block:: bash
+
+    schematic model -c /path/to/config.yml submit -mp <your csv manifest path> -d <your synapse top level folder id> -vc <your data type> -mrt file_only -no-fa -tcn "class_label"
+
+- **-c /path/to/config.yml**: Specifies the configuration file containing the data model location and asset view (`master_fileview_id`).
+- **-mp**: Your manifest file path.
+- **-mrt**: The format of manifest submission. The options are: "table_and_file", "file_only", "file_and_entities", "table_file_and_entities". "file_only" option would submit the manifest as a file.
+- **-vc <your_data_type>**: Defines the data type/schema model for the manifest (e.g., `"Patient"`, `"Biospecimen"`).
+- **-d <your_dataset_id>**: Retrieves the existing manifest associated with a specific dataset on Synpase.
+- **-fa**: Enable file annotations upload.
+- **-tcn**: Table Column Names: This is optional, and the available options are "class_label", "display_label", and "display_name". The default is "class_label", but you can change it based on your requirements.
+
+
+Option 2: Use the API
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+    During submission, validation is optional. If you have finished validation in previous step, you could skip validation by removing the default inputs.
+
+
+1. Locate the **model/submit** endpoint in the **Swagger UI**.
+2. Click **"Try it out"** to enable input fields.
+3. Enter the required parameters and execute the request:
+
+   - **schema_url**: The URL of your data model.
+     - If your data model is hosted on **GitHub**, the URL should follow this format:
+       - JSON-LD: `https://raw.githubusercontent.com/<your-repo-path>/data-model.jsonld`
+       - CSV: `https://raw.githubusercontent.com/<your-repo-path>/data-model.csv`
+
```

Comment:
I wonder if we should have a section somewhere that describes the data model options, and then link to it from here.

Reply:
I think we may have upcoming work to add in the documentation for the different data models.